src/status_im/contacts/screen.cljs

```diff
@@ -131,7 +133,7 @@
         (when @show-toolbar-shadow?
           [linear-gradient {:style  st/contact-group-header-gradient-bottom
                             :colors st/contact-group-header-gradient-bottom-colors}])]
-       (if (pos? (+ @dapps-count @people-count))
+       (if (pos? (+ @dapps-count @state-of-the-dapps-count @people-count))
```

Comment:
i'm reworking (and redesign) contacts now, so i'll do this change if you don't mind